src/entities/Votes/utils.ts

```diff
@@ -16,20 +18,38 @@ export function toProposalIds(ids?: undefined | null | string | string[]) {
   return list.filter((id) => isUUID(String(id)))
 }
 
-export function createVotes(votes: SnapshotVote[]) {
+function _createVotes<T>(votes: SnapshotVote[], returnValue: (vote: Vote, prevValue?: T) => T): Record<string, T> {
```

Comment:
This function could be named just `createVotes` as it is no longer clashing with the other function name.